content/en/docs/concepts/storage/volumes.md

```diff
@@ -249,43 +224,34 @@ spec:
 ```
 
 The `log-config` ConfigMap is mounted as a volume, and all contents stored in
-its `log_level` entry are mounted into the Pod at path `/etc/config/log_level`.
+its `log_level` entry are mounted into the pod at path `/etc/config/log_level`.
 Note that this path is derived from the volume's `mountPath` and the `path`
 keyed with `log_level`.
 
-{{< note >}}
-* You must create a [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/)
-  before you can use it.
-
-* A container using a ConfigMap as a [`subPath`](#using-subpath) volume mount will not
-  receive ConfigMap updates.
-
-* Text data is exposed as files using the UTF-8 character encoding. For other character encodings, use `binaryData`.
-{{< /note >}}
+Text data is exposed as files using UTF-8 character encoding. To use some other character encoding, use binaryData.
```

Comment:
```suggestion
Values from ConfigMaps are exposed as text files, using UTF-8 character encoding. To use some other character encoding, or to store information that is not text, use the `binaryData` (rather than `data`) field in the ConfigMap.
```
Something like that? Could save rewording this for another PR, BTW.